frontend/src/components/TableView/TableView.tsx

```diff
@@ -40,6 +42,7 @@ export const TableView = <T extends MRT_RowData>({
   title?: string
   data: T[] | undefined
   columns: MRT_ColumnDef<T>[]
+  hiddenColumns: MRT_VisibilityState
```

Comment:
Should this be actually renamed to visibleColumns or similar. The logic is the "wrong" way around: `hiddenColumns = { id: false }` and not `id: true`.

Reply:
Makes sense. Will do